src/rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -276,7 +281,7 @@ docker run --name $docker_name \
   --volume $container_local_dir/$bootstrap_dir:/pai/bootstrap:ro \
   --volume $container_local_dir/$code_dir:/pai/code:ro \
   --volume /var/drivers/nvidia/current:/usr/local/nvidia:ro \
-  --volume /etc/hadoop-configuration-for-jobs:/etc/hadoop:ro \
+  --volume $container_local_dir/$hadoop_config_dir:/etc/hadoop \
   --volume $container_local_dir/$ssh_config_file:/pai/ssh_config/ \
```

Comment:
Just noticed this line of code, should we make /pai/ssh_config also readonly @wangcan0329?

Reply:
it makes sense to mount it as readonly.

Reply:
Sure. It's better to mount it as readonly, I will fix it.